src/app/core/services/authentication/authentication.service.ts

```diff
@@ -47,7 +47,7 @@ export class AuthenticationService {
   }
 
   signUp(name: string, username: string, email: string, password: string){
-    return this.apiService.post(routes.signup, { name, username, email, password})
+    return this.apiService.post(routes.signup, { name, username, email, password, url: window.location.href})
```

Comment:
window.location.href return full current URL, I think should pass domain only

Reply:
Yes, the full current url is exactly what I want to pass to the sign up api for the return URL, because the backend requires that and always be handled to get host name